protzilla/run.py

```diff
@@ -178,6 +178,10 @@ def current_plots(self) -> Plots | None:
     @property
     def current_outputs(self) -> Output:
         return self.steps.current_step.output
+    
+    @property
+    def current_filtered_data(self) -> Output:
+        return self.steps.current_step.datatable_filtered_data
```

Comment:
Hier würde ich datatable rauslassen, damit es konsistent zu den anderen Benennungen ist? 
```suggestion
        return self.steps.current_step.filtered_data
```

Reply:
Oder vllt filtered_datatable damit der Name etwas kürzer ist aber trotzdem noch beschreibt, dass die Daten nie zum Weiter-Berechnen verwendet werden sondern nur in der Anzeige-Tabelle